docs/CHANGELOG.rst

```diff
@@ -6,6 +6,21 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+=================
+Unreleased
+=================
+
+Changed
+-------
+- Add bamclipper tool to ``resolwebio/dnaseq:4.3.0`` image
```

Comment:
Bamclipper requires `parallel` package to run. Append this to `resolwebio/dnaseq` Docker installation files.

Reply:
`parallel` come shipped with R (since 2.14.0) so I'm not sure this is really needed.

Reply:
Try to run the tool in locally build image to test this and be sure.

Reply:
Mea culpa, I now see you were referring to the linux's `parallel` package, not R's.

Reply:
Added parallel package to ubuntu installation.